tests/frontends/gpytorch/gpr_torch.py

```diff
@@ -4,7 +4,7 @@
 import polyscope as ps
 import torch
 
-from geometric_kernels.frontends import GPytorchGeometricKernel
+from geometric_kernels.frontends.gpytorch import GPytorchGeometricKernel
```

Comment:
We really need some meta wrapper that chooses a specific kernel based on the backend, to avoid unwieldy imports but that could be saved for later.

Reply:
Yeah, I agree.